miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppStorage.kt

```diff
@@ -76,4 +76,7 @@ internal class MiniAppStorage(
             }
         }
     }
+
+    fun isMiniAppVersionExisted(appId: String, versionId: String): Boolean =
+        File(getMiniAppVersionPath(appId, versionId)).exists()
```

Comment:
Maybe this could just be moved inside the `isVersionDownloaded` function? Because the Mini App shouldn't be counted as downloaded if the files don't exist, right?